docs/schema/schema.scaffold.ts

```diff
@@ -57,8 +57,34 @@ type Question = {
    * group is an optional key that can be used to group questions together in a shared view.
    * */
   group?: string;
+  validate?: Validator;
 };
 
+type Validator = {
+  /**
+   * required (when true) will ensure that the user provides a value for the prompt. Only some prompt types support this option.
+   * */
+  required?: boolean;
+  match?: {
+    /**
+     * regex is a regular expression that the user's input must match.
+     */
+    regex: string;
+    /**
+     * message is a message that will be displayed to the user if the input does not match the pattern. Only some prompt types support this option.
+     * */
+    message?: string;
+  }
+  /**
+   * min is the minimum value that the user's input must be greater than or equal to.
```

Comment:
clarify that if the input type is a string, this is the minimum length of the string, if it's a list, it's the minimum number of items in the list, and what about other datatypes?